data/scripts/sol-flow-node-type-gen.py.in

```diff
@@ -783,7 +817,10 @@ static void
         outfile.write("    if (%s.packet_type == NULL) {\n" % (all_ports_names[0]))
 
         for port, d_type in zip(all_ports_names, all_ports_types):
-            packet_type = packet_type_from_data_type(d_type)
+            if d_type.startswith("composed:"):
```

Comment:
minor, but why not `is_composed(d_type)` and then `get_composed_types_with_underscore(d_type)`?